tasks/main.yaml

```diff
@@ -26,5 +26,6 @@
   when: system.features.sudoers
 - import_tasks: authorized_keys.yaml
   when: system.features.authorized_keys
+- import_tasks: ssh_private_key.yaml
```

Comment:
Missing feature flag.

Reply:
If no private key is set this task does nothing, so a feature flag would not be necessary, otherwise if a private key is set, and you forget to set the feature flag will lead to confusion, why it is not working.